homework_1/ex2.py

```diff
@@ -0,0 +1,5 @@
+S = int(input("Введите общее количество журавликов: "))
+Pete = S//6
+Sergey = Pete
+Kate = 4*Pete
+print("Петя сделал {} журавликов, Катя сделала {} журавликов, Сережа - {} журавликов".format(Pete, Kate, Sergey))
```

Comment:
В Python обычно не принято начинать имена переменных функций и так далее с символов в верхнем регистре. Исключение составляют имена классов.
```suggestion
Pete = Sergey  = S // 6
Kate = 4 * Pete
print(f"Петя сделал {Pete} журавликов, Катя сделала {Kate} журавликов, Сережа - {Sergey} журавликов")
```